aea/cli/run.py

```diff
@@ -93,9 +93,12 @@ def run(click_context, connection_name: str, env_file: str, install_deps: bool):
     _try_to_load_agent_config(ctx)
     _load_env_file(env_file)
     agent_name = cast(str, ctx.agent_config.agent_name)
+
     _verify_or_create_private_keys(ctx)
     wallet = Wallet(dict([(identifier, config.path) for identifier, config in ctx.agent_config.private_key_paths.read_all()]))
 
+    _verify_ledger_apis_access(ctx)
```

Comment:
Here we verify that we can actually connect to the ledgers specified via configs